### ISSUE TITLE
Ensuring article.create() modal 

### ISSUE THREAD

**Issue by QuietEvolver** [EwRicklefs/Watch-the-screens#41]
@CPJanz Hi Carl, was wondering if the reporter's modal for article.create() submit on'click' was going to be the same cross-over template for the admin.  While this seems linear, I haven't known of any specs regarding this from discussions, many thanks.

**Comment by QuietEvolver** [comment]
@CPJanz: Since the Admin.js is up, the question is reversed: will this be the same for the reporter?(seems like less coding); 

Please feel free to close this project 'Question' after answered; thanks, Carl!

**Comment by CPJanz** [comment]
@QuietEvolver I'm not sure I understand the question. When a reporter submits an article we'll create a new row in the article db. The admin page won't have an article create route but will have a get route hitting articles to populate the article dropdown and an update route to change flag an article as hidden.

Let me know if this answers your question

**Comment by QuietEvolver** [comment]
@CPJanz I hadn't been privy to any of that.  This is pretty critical info.  From your response, it seems the admin (*who had formerly had CRU(no D)) has now been assigned different functionality.   Question: Do you by any chance know if the breaking news/new article will hinge solely on the reporter?  I want to be sure to deliver precisely what the team needs as we are halfway to launch.  Thank you for  all your help Carl.

**Comment by CPJanz** [comment]
The admin should have the same functionality as laid out in the user stories we wrote on Friday. They are able to update articles by changing the is_hidden boolean to true but that is the extent of their involvement  with articles. 

For the breaking news icon, we had talked about using a client side variable which would switch states based on whether or not the article had been seen in the article view before. It looks like we captured some of that discussion in the article story but I'll create a new story to full encapsulate that work.
